packages/app/src/app/components/Preview/DevTools/React-Devtools/elements.ts

```diff
@@ -8,6 +8,12 @@ export const Container = styled.div`
   color: ${props =>
     props.theme['editor.foreground'] || 'rgba(255, 255, 255, 0.8)'};
 
+  /* do not show button because it doesn't work */
+  button[class^='ToggleOff___'],
```

Comment:
Whohh what is this?

Reply:
You are right, it works on Codesanbdox but not locally